app/serializers/discourse_post_event/event_serializer.rb

```diff
@@ -129,8 +131,50 @@ def should_display_invitees
       (object.public? && object.invitees.count > 0) ||
         (object.private? && object.raw_invitees.count > 0)
     end
+
     def category_id
       object.post.topic.category_id
     end
+
+    def include_next_dates?
+      object.recurrence
```

Comment:
I think you could add a nice `recurring?` method on `DiscoursePostEvent::Event` model that just does `object.recurrence.present?`, then you can call it like this:

```suggestion
      object.recurring?
```